content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,128 @@
+---
+reviewers:
+- sig-api-machinery
+- sig-architecture
+- sig-cli
+- sig-cluster-lifecycle
+- sig-node
+- sig-release
+title: Kubernetes Version and Version Skew Support Policy
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+This document describes the maximum version skew supported between various Kubernetes components.
```

Comment:
Does "version skew" need more information, or an example? I'm not sure this will connect with people who are k8s users / admins but not developers. Is this guidance more for them, or more for k8s devs to understand where their work should be going?

Reply:
added a link from the cluster operator > foundational section